tests/ZfcRbacTest/Factory/AuthorizationServiceDelegatorTest.php

```diff
@@ -75,7 +75,7 @@ public function testAuthorizationServiceIsInjectedWithDelegator()
         $serviceManager = ServiceManagerFactory::getServiceManager();
 
         if (method_exists($serviceManager, 'build')) {
-            $this->markTestSkipped('this test is only vor zend-servicemanager v2');
```

Comment:
Might be possible to remove tests/support for servicemanager v2 -- I can't imagine people are going to want to move to Laminas without upgrading first. Of course, maybe I'm wrong! Probably not something you want to do in this PR, of course, but an idea for the future.